src/main/webapp/app/exercises/shared/included-in-overall-score-picker/included-in-overall-score-picker.component.ts

```diff
@@ -11,6 +11,9 @@ export class IncludedInOverallScorePickerComponent {
 
     @Input()
     includedInOverallScore: IncludedInOverallScore | undefined;
+    // Option to hide the NOT_INCLUDED option (used for exam exercises)
+    @Input()
+    excludeNotIncluded: boolean;
```

Comment:
`excludeNotIncluded` sounds a bit strange to me, maybe `allowNotIncluded` and invert the usage?